inst/NEWS.Rd

```diff
@@ -7,6 +7,8 @@
   \itemize{
     \item Changes in Rcpp API:
     \itemize{
+      \item XPtr gains a parameter \code{finalizeoOnExit} to enable running the
```

Comment:
May also move the entry to the bottom.  They are mostly chronological.  In any even I can clean up both when finalizing the release "real soon now".